ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala

```diff
@@ -380,4 +393,41 @@ class SSLConfigParser(c: EnrichedConfig, classLoader: ClassLoader) {
 
     TrustManagerConfig(algorithm, trustStoreInfos)
   }
+
+  def parseSSLParameters(config: EnrichedConfig): SSLParametersConfig = {
+    // could instanciate SSLParameters directly, but seems less clean, here we only parse config
```

Comment:
note this note; so technically this could return `SSLParameters` directly, but it felt like mixing "setting up" with "configuration", so I introduced this Config